### ISSUE TITLE
Beta ember-cli-babel versions trigger error

### ISSUE THREAD

**Issue by jrjohnson** [salsify/ember-cli-dependency-lint#5]
I'm seeing:
```
$ ember dependency-lint
ember-cli-babel
Allowed: *
Found: 5.2.4, 6.0.0-beta.5, 5.1.10
```

As https://github.com/machty/ember-maybe-import-regenerator is now requiring the beta bower version.

**Comment by kturney** [comment]
Just ran into this as well.

This is apparently the semver lib [working as designed](https://github.com/npm/node-semver/issues/162).

I worked around the issue with:
```js
// config/dependency-lint.js

module.exports = {
  allowedVersions: {
    'ember-cli-babel': '* || >=6.0.0-beta.1'
  }
};
```

**Comment by dfreeman** [comment]
Ran into this about an hour ago as well. @kturney's suggestion will work as a workaround for the moment, but I'm leaning toward special-casing a version constraint of `'*'` within this addon to bypass `semver.satisfies` completely and just truly accept anything.

Seem reasonable?

**Comment by kturney** [comment]
@dfreeman seems reasonable to me

**Comment by jrjohnson** [comment]
Yes. I think * should mean *. Sounds good.

**Comment by dfreeman** [comment]
Fixed in v1.0.2 — thanks for reporting!
